packages/core/src/workspace/skills/workspace-skills.test.ts

```
@@ -2170,4 +2170,217 @@ Premium instructions.
+        new Promise<void>(resolve => {
+          trackConcurrency();
+          setTimeout(() => {
+            tracker.ioCalls++;
+            tracker._inflight--;
+            resolve();
+          }, delayMs);
+        });
+
+      return {
+        ...fast,
+        ioCalls: 0,
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)